src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -546,8 +546,9 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
     {
         if (_transitionToState(ConnectionState::Closing))
         {
-            // EXIT POINT
-            _clientExitWait.reset(); // immediately stop waiting for the client to exit.
+            // _clientExitWait holds a CreateThreadpoolWait() which holds a weak reference to "this".
```

Comment:
```suggestion
            // EXIT POINT

            // _clientExitWait holds a CreateThreadpoolWait() which holds a weak reference to "this".
```

please keep this comment; it is an important landmark documented at the top of the file